pkg/client/localsubjectaccessreview.go

```diff
@@ -63,7 +63,7 @@ func (c *localSubjectAccessReviews) Create(sar *authorizationapi.LocalSubjectAcc
 		}
 		deprecatedResponse := &authorizationapi.SubjectAccessReviewResponse{}
 
-		req, err := overrideAuth(c.token, c.r.Post().Namespace(c.ns).Resource("subjectAccessReviews"))
+		req, err = overrideAuth(c.token, c.r.Post().Namespace(c.ns).Resource("subjectAccessReviews"))
```

Comment:
So I thought this created a new `req` variable in this scope instead of re-using the one from above.  I don't like re-using variables.  If we want to remove the marker, you could change it to a different variable name?